wdl2cwl/tests/cwl_files/BuildCembaReferences.cwl

```diff
@@ -0,0 +1,395 @@
+          - class: InlineJavascriptRequirement
+          - class: NetworkAccess
+            networkAccess: true
+          - class: ResourceRequirement
+            coresMin: 1
+            ramMin: 3337.860107421875
+            outdirMin: 1024
+        cwlVersion: v1.2
+        baseCommand:
+          - bash
+          - example.sh
+  - id: IndexForward
+    in:
+      - id: fasta_input
+        source: IndexForward.fasta_input
+      - id: index_prefix
+        source: IndexForward.index_prefix
+      - id: monitoring_script
+        source: IndexForward.monitoring_script
+    out:
+      - id: bowtie2_index_files
+      - id: monitoring_log
+    run:
+        class: CommandLineTool
+        id: Bowtie2Build
+        inputs:
+          - id: fasta_input
+            type: File
+          - id: index_prefix
+            type: string
+          - id: monitoring_script
+            type:
+              - File
+              - 'null'
+        outputs:
+          - id: bowtie2_index_files
+            type:
+                items: File
+                type: array
+            outputBinding:
+                glob: $(inputs.index_prefix + "*")
+          - id: monitoring_log
+            type: File
+            outputBinding:
+                glob: monitoring.log
+        requirements:
+          - class: DockerRequirement
+            dockerPull: quay.io/broadinstitute/bowtie2:2.3.4.3
+          - class: InitialWorkDirRequirement
+            listing:
+              - entryname: example.sh
+                entry: |4
+
+                    # if the WDL/task contains a monitoring script as input
+                    if [ ! -z "$(inputs.monitoring_script === null ? "" : inputs.monitoring_script.path)" ]; then
+                      chmod a+x $(inputs.monitoring_script === null ? "" : inputs.monitoring_script.path)
+                      $(inputs.monitoring_script === null ? "" : inputs.monitoring_script.path) > monitoring.log &
+                    else
+                      echo "No monitoring script given as input" > monitoring.log &
+                    fi
+
+                    bowtie2-build \
+                      -f $(inputs.fasta_input.path) $(inputs.index_prefix)
+          - class: InlineJavascriptRequirement
+          - class: NetworkAccess
+            networkAccess: true
+          - class: ResourceRequirement
+            coresMin: 1
+            ramMin: 6675.72021484375
+            outdirMin: 1024
+        cwlVersion: v1.2
+        baseCommand:
+          - bash
+          - example.sh
+  - id: IndexReverse
+    in:
+      - id: fasta_input
+        source: IndexReverse.fasta_input
+      - id: index_prefix
+        source: IndexReverse.index_prefix
+      - id: monitoring_script
+        source: IndexReverse.monitoring_script
+    out:
+      - id: bowtie2_index_files
+      - id: monitoring_log
+    run:
+        class: CommandLineTool
+        id: Bowtie2Build
```

Comment:
In another PR we should only convert a WDL task once, and store it in the `$graph`, then we can `run: #Bowtie2Build` and not have to repeat ourselves.

Later we can even write these to separate `.cwl` files, if a command line option is given to do so